### PR TITLE
Factor out common label animations.

### PR DESCRIPTION
**Pull request by alantreadway**

### CODE REVIEW

File: packages/ag-charts-community/src/chart/series/seriesLabelUtil.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    labelSelections: Selection<NodeWithOpacity, T>[]
+) {
+    const duration = animationManager.defaultDuration;
+    staticFromToMotion(
+        `${id}_empty-update-ready_labels`,
+        animationManager,
+        labelSelections,
+        { opacity: 0 },
+        { opacity: 1 },
+        { delay: duration, duration: animationManager.defaultDuration * 0.2 }
```
Comment (lsjroberts) [inline packages/ag-charts-community/src/chart/series/seriesLabelUtil.ts]: Current requirements put this as hardcoded to 200ms, regardless the duration of the rest of the animation. Pending further expansion of https://ag-grid.atlassian.net/browse/AG-9189

**Review by lsjroberts — APPROVED** [review]
lgtm

File: packages/ag-charts-community/src/chart/series/seriesLabelUtil.ts (hunk 1 above)
Comment (alantreadway) [reply packages/ag-charts-community/src/chart/series/seriesLabelUtil.ts]: Ok - I'll switch back 👍

Comment (alantreadway) [reply packages/ag-charts-community/src/chart/series/seriesLabelUtil.ts]: Fixed - thanks for pointing that out

**Pull request merged**